pkg/sfu/downtrack.go

```diff
@@ -1056,10 +1056,6 @@ func (d *DownTrack) retransmitPackets(nacks []uint16) {
 	nackMisses := uint32(0)
 	for _, meta := range d.sequencer.getPacketsMeta(filtered) {
 		if meta.layer == int8(InvalidLayerSpatial) {
-			if meta.nacked > 1 {
```

Comment:
Ignoring NACKs of padding packets. We do not retransmit padding. So, do  not take signal of NACKs on that. Had done this before as clients stop asking for those padding packets if it can proceed with the decoding. But, I think it is unnecessary to include these.